pom.xml

```diff
@@ -343,7 +335,9 @@
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.4</version>
           <configuration>
-            <quiet>true</quiet>
```

Comment:
I guess I would have left that as-is, in case for instance plugin have overridden/forced the execution of the javadoc plugin. Here they would be back to the verbose mode.
As we are here in the `pluginManagement` part, we should stay in quiet mode by default IMO: because it makes sense, and because that was the current value and changing the behaviour for inheriting plugins does not look justified to me.

Reply:
Also, currently that would make javadoc back to non-quiet during releases. I am against this change without justification :bug: 

https://github.com/jglick/plugin-pom/blob/043a10d6eb0fc4008a7ccafacf5ca407e5537e83/pom.xml#L803-L839 is the place where the plugin is configured, FWIW for others (and for me for possible future ref :)).

Reply:
I switched it back to verbose (rather, non-quiet) mode because that provides better opportunities for debugging and my understanding of the rationale for `-quiet` was to avoid log noise during regular developer or CI builds, which is less of an issue when the Javadoc goal is run only during releases (incl. incremental ones). However I do not have strong feelings about it either way, so I will revert this aspect to unblock. I suppose actual errors will be displayed in any case.